packages/host/app/components/operator-mode/code-submode/playground/playground-content.gts

```diff
@@ -484,15 +529,16 @@ export default class PlaygroundContent extends Component<Signature> {
     await this.cardResource?.loaded; // TODO: remove await when card-resource is refactored
     if (this.card) {
       this.recentFilesService.addRecentFileUrl(`${this.card.id}.json`);
-      if (this.args.isFieldDef) {
-        this.createNewField.perform(this.card as Spec);
-      } else {
-        this.persistSelections(this.card.id, 'edit'); // open new instance in playground in edit format
-      }
+      this.persistSelections(
+        this.card.id,
+        'edit',
+        this.args.isFieldDef ? 0 : undefined,
+      ); // open new instance in playground in edit format
     }
+    this.closeInstanceChooser();
   });
 
-  private createNewField = task(async (specCard: Spec) => {
+  private createNewField = restartableTask(async (specCard: Spec) => {
```

Comment:
this is now a `restartableTask` because `fieldInstances` getter is called twice and this task starts running if there are no instances